### PR TITLE
Add bio for Fgondim

### LINKED ISSUE #10: Add bio for fgondim
- [x] Create a branch
- [x] Add a bio file for fgondim
- [x]  Open a pull request
- [x] Give & receive feedback
- [x] Merge pull request

### PR DESCRIPTION
**Pull request by fgondim**
This fixes #10 by creating a new bio file for fgondim

### CODE REVIEW

File: fgondim.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Fgondim
```
Comment (ghost) [inline fgondim.md]: Maybe this should have your username ?

File: fgondim.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+* Name: Flavio
+* From: Brazil
+* Location: Montreal, Canada
```
Comment (autodesk-delphine) [inline fgondim.md:5]: Where in Montreal?

Comment (fgondim) [inline fgondim.md:5]: Prince Arthur - Saint-Laurent

File: fgondim.md (hunk 1 above)
Comment (fgondim) [inline fgondim.md]: better?

**Comment by autodesk-delphine** [comment]
Perfect @fgondim :+1:

**Pull request merged**